### DISCUSSION TITLE
Auto imports naming: Plural for directory and singular for component name.

### DISCUSSION THREAD

**Discussion by mfazail** [nuxt/nuxt#23687] [Ideas]
Can we have plural name for directory but component should be singular while using auto import, for example - 

```bash
/components
    /Vehicles        // plural
        /Car.vue
        /Bike.vue
```

then in vue files

```vue
...
<VehicleCar /> // singular 
<VehicleBike /> // singular
...

```

I guess this would feel more natural when reading component name.
Thank you 💚 nuxt for great DX.

**Comment by TheAlexLichter** [comment]
You can define your own structure, including the prefix used, via your `nuxt.config.ts`.

See:

* https://nuxt.com/docs/guide/concepts/auto-imports#auto-imported-components
* https://youtu.be/FT2LQJ2NvVI?t=70
